src/main/java/net/oauth/jsontoken/AbstractJsonTokenParser.java

```diff
@@ -60,6 +60,7 @@
    *
    * @param tokenString The original encoded representation of a JWT
    * @return Unverified contents of the JWT as a JsonToken
+   * @throws IllegalStateException when tokenString isn't in three parts
```

Comment:
nit: "when tokenString is not a properly formatted JWT" here and other methods.

Reply:
Changed